2016-09/september-21.md

```diff
@@ -0,0 +1,49 @@
+## September 21 ([discuss](https://github.com/webpack/meeting-notes/pull/14))
+
+### Attendees
+
+* [Tobias Koppers](http://github.com/sokra)
+* [Sean Larkin](http://github.com/thelarkinn)
+* [Juho Vepsäläinen](http://github.com/bebraw)
+
+
+### Agenda
+
+#### Documentation Updates
+
+* Juho is going to start working on [antwar/antwar#103](https://github.com/antwarjs/antwar/issues/103) next week.
```

Comment:
I'll bet that @KyleAMathews would be happy to give back to Webpack by submitting a PR to do this with [gatsby](https://github.com/gatsbyjs/gatsby), or @MoOx would be happy to do so with [phenomic](https://github.com/MoOx/phenomic). :)